README.md

```diff
@@ -21,6 +21,7 @@ An updated list of entities and triggers with their respective explanation of wh
 ### Entities
 - **Bubble Columns**: Streams of bubbles which push the player and other objects around.
 - **Flag Kill Barrier**: A barrier that will kill you if you have the flag assigned to it.
+- **Sticky Jellyfish**: A Jellyfish that will levitate once the player drops it or it loses its horizontal speed.
```

Comment:
It's capitalized as is everything else since it's the name, if it's different in Ahorn probably should change it to match all the names in the readme